BedrockCommand.cpp

```diff
@@ -196,8 +196,21 @@ void BedrockCommand::finalizeTimingInfo() {
         }
     }
 
+    // This is a hack to support Auth's old `Get` format where we have a `returnValueList` of items to return rather
+    // than a specific name. The timing profile of every version of this command is wildly different and it's impossible
+    // to reason about which ones cause performance issues when they're all globbed together.
+    // In the future, let's find a better way to do this. For now, this gets us the data we need.
+    string methodName = request.methodLine + (request.isSet("returnValueList") ? "_" + request["returnValueList"] : ""s);
```

Comment:
The returnvaluelist is a list of an arbitrary number of return values, yes? That means that for each bedrock host, we're potentially creating dozens of new metrics. Wondering if there's a better way. I'll sleep on it.